examples/tutorials/advanced/cartesian_histograms.py

```diff
@@ -0,0 +1,340 @@
+)
+
+# Shift anchor point 11 centimeters to the right
+fig.shift_origin(xshift="11c")
+
+# Create histogram for data02 showing frequency percent
+fig.histogram(
+    region=[0, 200, 0, 100],
+    projection="X10c",
+    frame=["lSnE", "xaf10", "ya10f5+lFrequency percent"],
+    data=data02,
+    series=10,
+    fill="orange",
+    pen="1p,darkgray",
+    # Choose frequency percent via the "histtype" parameter
+    histtype=1,
+)
+
+fig.show()
+
+
+###############################################################################
+# Cumulative values
+# -----------------
+# To create a histogram showing the cumulative values set ``cumulative=True``.
+# Here, the bars of the cumulative histogram are filled with a pattern via
+# the ``fill`` parameter. Annotate each bar with the counts it represents
+# using the ``annotate`` parameter.
+
+# Create new figure instance
+fig = pygmt.Figure()
+
+# Create histogram for data01 showing the counts per bin
+fig.histogram(
+    region=[0, 200, 0, 43],
+    projection="X10c",
+    frame=["WSne", "xaf10", "ya5f1+lCounts"],
+    data=data01,
+    series=10,
+    fill="red3",
+    pen="1p,darkgray",
+    histtype=0,
+    # Annotate each bar with the counts it represents
+    annotate=True,
+)
+
+# Shift anchor point 11 centimeters to the right
+fig.shift_origin(xshift="11c")
+
+# Create histogram for data01 showing the cumulative counts
+fig.histogram(
+    region=[0, 200, 0, 43],
+    projection="X10c",
+    frame=["wSnE", "xaf10", "ya5f1+lCumulative counts"],
+    data=data01,
+    series=10,
+    # Use pattern (p) number 8 as fill for the bars
+    # Set the background (+b) to white [Default]
+    # Set the foreground (+f) to black [Default]
+    fill="p8+bwhite+fblack",
+    pen="1p,darkgray",
+    histtype=0,
+    # Show cumulative counts
+    cumulative=True,
+    # Offest (+o) the label by 10 points in negative y-direction
+    annotate="+o-10p",
+)
+
+fig.show()
+
+
+###############################################################################
+# Overlaid bars
+# -------------
+# Overlaid or overlapping bars can be achieved by plotting two or serveral
+# histograms, each for one data set, on top of each other. The legend entry
+# can be specified via the ``label`` parameter.
+#
+# Limitations of histograms with overlaid bars are:
+#
+# - Mixing of colors or/and patterns
+# - Visually more colors or/and patterns than data sets
+# - Visually a "third histogram" (or more in case of more than two data sets)
+
+# Create new figure instance
+fig = pygmt.Figure()
+
+# Create histogram for data01
+fig.histogram(
+    region=[0, 200, 0, 10],
+    projection="X10c",
+    frame=["WSne", "xaf10", "ya2f1+lCounts"],
+    data=data01,
+    series=10,
+    fill="red3",
+    pen="1p,darkgray",
+    histtype=0,
+    # Set legend entry
+    label="data01",
+)
+
+# Create histogram for data02
+# It is plotted on top of the histogram for data01
+fig.histogram(
+    region=[0, 200, 0, 10],
+    projection="X10c",
```

Comment:
`region` and `projection` are not needed because it's already set in previous `Figure.histogram` call.